README.md

```diff
@@ -13,6 +13,9 @@ The current features are:
 - (with extendability for new convergence layer adapters, routing algorithms, and storage managers)
 
 ## Getting Started
+
+Installation has been successfully tested on Windows and Linux (x86_64 and arm64). The required [mpy-cross](https://pypi.org/project/mpy-cross/) has been shown to have installation issues on Mac M1s under Mac OS X and may have to be built and installed manually.
+
```

Comment:
This is valuable information, but it may be better suited at the beginning of the "MicroPython Installation Guide" section.

Reply:
I've moved the paragraph to the beginning of the MicroPython section and additionally added placeholders in the `wlan.json` to make wifi configuration clearer.

Reply:
Thank you for the contribution. The placeholders for clarity are also a good idea.